src/04-modality-specific-files/10-functional-near-infrared-spectroscopy.md

```diff
@@ -0,0 +1,385 @@
+Any of the channel types defined in other BIDS specification can be used here as well such as ACCEL or MAGN.
+As several of these data types are commonly acquired using fNIRS devices they are included as an example at the base of the table.
+Note that upper-case is REQUIRED.
+
+| **Keyword**                 | **Description**                                                                                                                                                              |
+| --------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| NIRSCWAMPLITUDE             | Continuous wave amplitude measurements. Equivalent to dataType 001 in SNIRF.                                                                                                 |
+| NIRSCWFLUORESCENSEAMPLITUDE | Continuous wave fluorescence amplitude measurements. Equivalent to dataType 051 in SNIRF.                                                                                    |
+| NIRSCWOPTICALDENSITY        | Continuous wave change in optical density measurements. Equivalent to dataTypeLabel dOD in SNIRF.                                                                            |
+| NIRSCWHBO                   | Continuous wave oxygenated hemoglobin (oxyhemoglobin) concentration measurements. Equivalent to dataTypeLabel HbO in SNIRF.                                                  |
+| NIRSCWHBR                   | Continuous wave deoxygenated hemoglobin (deoxyhemoglobin) concentration measurements. Equivalent to dataTypeLabel HbR in SNIRF.                                              |
+| NIRSCWMUA                   | Continuous wave optical absorption  measurements. Equivalent to dataTypeLabel mua in SNIRF.                                                                                  |
+| ACCEL                       | Accelerometer channel, one channel for each orientation. An extra column `component` for the axis of the orientation MUST be added to the `_*channels.tsv` file (x, y or z). |
+| GYRO                        | Gyrometer channel, one channel for each orientation. An extra column `component` for the axis of the orientation MUST be added to the `_*channels.tsv` file (x, y or z).     |
+| MAGN                        | Magnetomenter channel, one channel for each orientation. An extra column `component` for the axis of the orientation MUST be added to the `_*channels.tsv` file (x, y or z). |
+| MISC                        | Miscellaneous                                                                                                                                                                |
+
+Example:
+
+```Text
+Name         type                   source      detector      wavelength_nominal   units
+S1-D1        NIRSCWAMPLITUDE        A1          Fz            760                  V
```

Comment:
how does one convert from the unit `V` (from the ADC) to the amount of light (i.e., the amount of photons) in `Lumen`, which can be converted to optical densities? In case `V` is specified, there should also be a detector-specific calibration value somewhere.

Reply:
In practice, just the raw voltage from the ADC is used, and no conversion to Lumen is performed. Optical density is performed on the raw ADV voltages (possibly after filtering etc).

This is discussed in  Huppert TJ, Diamond SG, Franceschini MA, Boas DA. HomER: a review of time-series analysis methods for near-infrared spectroscopy of the brain. Appl Opt. 2009 Apr 1;48(10):D280-98. doi: 10.1364/ao.48.00d280. 
> In general, the detected light intensity is proportional to the amplitude of the voltage signal reported by the photon detectors, assuming that the instability and possible nonlinearity of the photon detectors of the NIRS instrument used can be neglected. By normalizing the intensity measurements to the incident light at the start of the experiment (in practice, usually the mean of the signal over the course of the study is used), one can neglect detector efficiency, detector analog amplification (gain), and the instrument’s absolute incident laser power, since these appear as scaling factors that multiply both the numerator and denominator

However, I can imagine future uses where precise knowledge of the Lumen is useful.
I just checked the SNIRF spec and this conversion parameter does not exist (someone correct me if I missed it, I was hoping to find a consistent naming).
We could add an OPTIONAL column called `calibration_lumen` that describes `The calibration required to convert from the channel units to Lumen`. What do you think @robertoostenveld @dboas @lpollonini ?

Reply:
Agreed that V (or even integer AD values) is sufficient in the subsequent analysis. I suggest not to add a calibration column to the `channels.tsv`, the calibration should be performed on the float values represented in the low-level SNIRF file. The specification of `units` here as you suggest is fine and corresponds to  [dataUnit](https://github.com/fNIRS/snirf/blob/v1.1/snirf_specification.md#nirsidatajmeasurementlistkdataunit).

Reply:
Thanks for the quick feedback @robertoostenveld 
I am marking this as resolved. Anyone who wants to reopen it please tag me so I get a notification (this PR is long!!)